src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1266,4 +1266,13 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
-</root>
+  <data name="ConsiderPassingBaseTypesAsParametersDescription" xml:space="preserve">
```

Comment:
I have tried to copy the description from fxcop but the title and message are customized. Let me know what's your preference.